config/routes.rb

```diff
@@ -1,6 +1,14 @@
 Rails.application.routes.draw do
   root "articles#index"
   resources :articles
+  resources :users
```

Comment:
I assumed you would just have one edit/update combination for both email and password, but since you don't have that, you will still need to have the only statement here otherwise you are creating routes that do not have corresponding endpoints in the controller.